sweepbatcher/sweep_batch.go

```diff
@@ -565,8 +586,48 @@ func (b *batch) Run(ctx context.Context) error {
 			timerChan = time.After(b.cfg.batchPublishDelay)
 			b.currentHeight = height
 
+			// Check if the batch became urgent.
+			timeout := b.timeout()
+			if timeout <= 0 {
+				b.log.Warnf("timeout=%v. Empty batch?", timeout)
```

Comment:
nit: logging is a bit vague, consider adding slightly more detail.

Reply:
Fixed. Now it logs this:

> Method timeout() returned %v. Number of sweeps: %d. It may be an empty batch.